Trimble.Modus.Components/Trimble.Modus.Components.csproj

```diff
@@ -17,17 +17,23 @@
 		<SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'tizen'">6.5</SupportedOSPlatformVersion>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <None Remove="Images\modus_close_icon.svg" />
+	</ItemGroup>
+
 	<ItemGroup>
 	  <EmbeddedResource Include="Images\black_close_icon.png" />
-	  <EmbeddedResource Include="Images\close_icon.png" />
-	  <EmbeddedResource Include="Images\modal_close_icon.png" />
 	  <EmbeddedResource Include="Images\blue_close_icon.png" />
 	  <EmbeddedResource Include="Images\helper_icon.png" />
 	  <EmbeddedResource Include="Images\input_error_icon.png" />
 	  <EmbeddedResource Include="Images\input_valid_icon.png" />
 	  <EmbeddedResource Include="Images\white_close_icon.png" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <MauiImage Include="Images\modus_close_icon.svg" />
```

Comment:
Can we use svg images in the coponents project? I remember Sam said, we cannot use them

Reply:
Yes, I was not able to view SVG images before but its working now. We need to set `build action = MauiImage` and use it like a normal PNG file.